internal/controller/pgupgrade/pgupgrade_controller.go

```diff
@@ -493,17 +494,15 @@ func (r *PGUpgradeReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		}
 
 		// Requeue to verify that Patroni endpoints are deleted
-		return ctrl.Result{Requeue: true}, err // FIXME
+		return runtime.RequeueWithBackoff(), err // FIXME
 	}
 
 	// TODO: write upgradeJob back to world? No, we will wake and see it when it
 	// has some progress. OTOH, whatever we just wrote has the latest metadata.generation.
 	// TODO: consider what it means to "re-use" the same PGUpgrade for more than
 	// one postgres version. Should the job name include the version number?
 
-	log.Info("Reconciled", "requeue", err != nil ||
-		result.Requeue ||
-		result.RequeueAfter > 0)
+	log.Info("Reconciled", "requeue", !result.IsZero() || err != nil)
```

Comment:
🤔 We declare `result` in the func, but where do we use it? Almost all the returns in this file are `return ctrl.Result{}, err`.

Reply:
Perhaps it's for L150, where we change `err` during a `defer`: https://github.com/CrunchyData/postgres-operator/blob/076c7ef6bc7f909e6f1c4cbc62a047155e578b7b/internal/controller/pgupgrade/pgupgrade_controller.go#L149-L150